src/screens/GroupInfo/elements/ModalAction.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import {Platform, StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 import Modal from 'react-native-modal';
+import SimpleToast from 'react-native-simple-toast';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `SimpleToast` from 'react-native-simple-toast' is a new addition and it seems to be used correctly in the code. It's being used to display a toast message when certain conditions are met.
```diff
+ import SimpleToast from 'react-native-simple-toast';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->